src/Uno.UI/UI/Xaml/Controls/AutoSuggestBox/AutoSuggestBox.cs

```diff
@@ -333,6 +337,11 @@ private void OnPopupOpened(object sender, object e)
 		{
 			LayoutPopup();
 		}
+g
+		private void OnSizeChanged(object sender, SizeChangedEventArgs args)
+		{
+			LayoutPopup();
```

Comment:
I think the correct thing is to set the Width on `SuggestionsContainer` to match WinUI

Reply:
Relevant WinUI code:

```cpp
//------------------------------------------------------------------------------
// Handler of the SizeChanged event.
//------------------------------------------------------------------------------
_Check_return_ HRESULT
AutoSuggestBox::OnSizeChanged(
    _In_ IInspectable* /*pSender*/,
    _In_ xaml::ISizeChangedEventArgs* /*pArgs*/)
{
    HRESULT hr = S_OK;

    if (m_tpSuggestionsContainerPart)
    {
        DOUBLE actualWidth = 0.;
        wrl::ComPtr<xaml::IFrameworkElement> spThisAsFE = this;

        IFC(spThisAsFE->get_ActualWidth(&actualWidth));
        IFC(m_tpSuggestionsContainerPart->put_Width(actualWidth));
    }

Cleanup:
    RRETURN(hr);
}

```